pennylane/workflow/qnode.py

```diff
@@ -599,16 +590,7 @@ def __copy__(self):
 
     def __repr__(self):
         """String representation."""
-        if isinstance(self.device, qml.devices.Device):
-            return f"<QNode: device='{self.device}', interface='{self.interface}', diff_method='{self.diff_method}'>"
-
-        detail = "<QNode: wires={}, device='{}', interface='{}', diff_method='{}'>"
-        return detail.format(
-            self.device.num_wires,
-            self.device.short_name,
-            self.interface,
-            self.diff_method,
-        )
+        return f"<QNode: device='{self.device}', interface='{self.interface}', diff_method='{self.diff_method}'>"

```

Comment:
I think we should have separate branches here for the facade since the repr will include `LegacyDeviceFacade` in it, which I don't like.